vela-templates/definitions/internal/policy/garbage-collect.cue

```diff
@@ -12,6 +12,8 @@ template: {
 		selector: #ResourcePolicyRuleSelector
 		// +usage=Specify the strategy for target resource to recycle
 		strategy: *"onAppUpdate" | "onAppDelete" | "never"
+		// +usage=Specify the deletion propagation strategy for target resource to delete
+		propagation?: "orphan" | "cascading"
```

Comment:
Which propagation will be used by default?

Reply:
The default case is having no propagation settings. For Kubernetes, when the propagation setting is absent, it will determine the propagation by resources. For example, for deployments, it will be "cascading" (or in other word "background"), but for jobs, it will be "orphan". It works like "auto".